### PR TITLE
Cosmetics

### PR DESCRIPTION
**Pull request by shamoya**

### CODE REVIEW

**Review by bekets — APPROVED** [review]

**Review by bekets — APPROVED** [review]

File: drivers/net/mlx5/mlx5_vdpa.c [hunk 1]
```diff
@@ -165,15 +165,34 @@ create_rq(struct vdpa_priv *priv, uint16_t qsize, uint16_t idx)
...
+ * 1 transmitq1
+ * ...
+ * 2(N-1) receiveqN
+ * 2(N-1)+1 transmitqN
+ * 2N controlq
+ */
+static bool is_virtq_recvq(int virtq_index, int nr_vring)
+{
+	if (virtq_index == 0 ||
+	    (virtq_index % 2 == 0 && virtq_index != nr_vring - 1))
```
Comment (shahafsh) [inline drivers/net/mlx5/mlx5_vdpa.c]: why the distinction between the first queue and the next ones?

it looks like Rx queue is every even queue which is not the last one (i.e. nr_vring)

**Review by shahafsh — COMMENTED** [review]
@shamoya 

small comment

File: drivers/net/mlx5/mlx5_vdpa.c (hunk 1 above)
Comment (shamoya) [reply drivers/net/mlx5/mlx5_vdpa.c]: Yes you are right, will change.
BTW @shahafsh, I tried moving the assert(nr_vring == 2) here and then just check virtq_index == 0, with 
TODO about changing the function when we have MQ.
But that wouldn't compiled with -Werror=unused-parameter  on the nr_vring.
Why is the compiler doesn't recognize the assert?

**Comment by shahafsh** [comment]
Asserts are only compile in debug mode.

Without the debug mode flag, the parameter is indeed unused.

**Pull request merged**